man/en/sos.1

```diff
@@ -10,8 +10,7 @@ sos \- A unified tool for collecting system logs and other debug information
 support representatives, and the like to assist in troubleshooting issues with
 a system or group of systems.
 
-The most well known function is \fB sos report\fR or \fBsos report\fR as it was
-previously known.
+The most well known function is \fB sos report\fR.
```

Comment:
minor nit, just the space before `sos`

Reply:
Fixed, and also fixed another reference that I think was incorrect.